processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorSubscription.java

```diff
@@ -124,13 +190,13 @@ private void waitForRemainingTasksCompletion(long timeoutMillis) {
             final int pendingTasksCount = contexts.totalPendingTasks();
             Duration elapsed = timer.duration();
             if (pendingTasksCount == 0) {
-                logger.debug("waiting for task completion is successful {} ns \\(^^)/",
-                             Utils.formatNanos(elapsed));
+                log.debug("waiting for task completion is successful {} ns \\(^^)/",
```

Comment:
[nits] Isn't it time to get rid of this ascii art?

Reply:
lol okay.

Reply:
a9e15c9 with some extra refactoring :)